webnn/resources/utils.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const ExecutionArray = ['async', 'sync'];
```

Comment:
I think I would simply inline the two arrays in webnn/idlharness

Reply:
Hi @dontcallmedom This `ExecutionArray` is also for operations tests, so I define it in this `utils.js` file.